xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml

```diff
@@ -122,28 +122,24 @@
       </TypeParameters>
       <Parameters>
         <Parameter Name="options" Type="System.Text.Json.JsonSerializerOptions" />
-        <Parameter Name="elementInfo" Type="System.Text.Json.Serialization.Metadata.JsonTypeInfo" />
-        <Parameter Name="numberHandling" Type="System.Text.Json.Serialization.JsonNumberHandling" />
-        <Parameter Name="serializeFunc" Type="System.Action&lt;System.Text.Json.Utf8JsonWriter,TElement[]&gt;" />
+        <Parameter Name="collectionInfo" Type="System.Text.Json.Serialization.Metadata.JsonCollectionInfoValues&lt;TElement[]&gt;" />
       </Parameters>
       <Docs>
-        <typeparam name="TElement">The generic definition of the element type.</typeparam>
-        <param name="options">The <see cref="T:System.Text.Json.JsonSerializerOptions" /> to use.</param>
-        <param name="elementInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the element type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <summary>Creates metadata for an array.</summary>
+        <typeparam name="TElement">To be added.</typeparam>
+        <param name="options">To be added.</param>
+        <param name="collectionInfo">To be added.</param>
+        <summary>To be added.</summary>
         <returns>To be added.</returns>
         <remarks>To be added.</remarks>
```

Comment:
The remarks for these methods in /// source should probably be imported, as they state:

"This API is for use by the output of the System.Text.Json source generator and should not be called directly."